txdbus/authentication.py

```diff
@@ -398,12 +405,12 @@ def _create_cookie(self, timefunc=time.time):
             if int(tpl[0]) >= cookie_id:
                 cookie_id = int(tpl[0]) + 1
 
-        cookie = binascii.hexlify(os.urandom(24))
+        cookie = binascii.hexlify(os.urandom(24)).decode()
```

Comment:
Again this, and the follow logic, would probably make more sense to be kept as `b"inary"` string